packages/@dcl/ecs/src/engine/jsx/types/global.d.ts

```diff
@@ -0,0 +1,35 @@
+import { DivProps } from '..'
+import { PBUiText } from '../../../components/generated/pb/UiText.gen'
```

Comment:
this is breaking the build. .d.ts should only import .d.ts, here it is pointing to a .ts